index.html

```diff
@@ -492,18 +501,6 @@ <h4 class="contact_head">Mubaris NK</h4>
   <!-- Custom JavaScript for this theme -->
   <script src="js/scrolling-nav.js"></script>
 
-
-
-
-
-
-  <script src="https://unpkg.com/vue"></script>
-  <script src="js/mentor.js"></script>
-  <script src="js/contestant.js"></script>
-  <script src="js/projects.js"></script>
-  <script src="js/issue-loader.js"></script>
```

Comment:
You deleted this script (probably by mistake) and issue loader does not work in your fork. Please check this that you don't delete thing not related to PR. :)

Reply:
made both requested changes, sorry for the trouble, merge?